lib/utils/fs.go

```diff
@@ -323,6 +323,11 @@ func removeSecure(filePath string, fi os.FileInfo) error {
 				}
 			}
 		}
+		// The file should be closed before removing it on Windows.
+		err := f.Close()
+		if err != nil {
+			return trace.Wrap(err)
+		}
 		return trace.ConvertSystemError(os.Remove(filePath))
```

Comment:
An error on `Close` doesn't mean that the file wasn't closed, so `os.Remove` might still work.
```suggestion
		// The file should be closed before removing it on Windows.
		return trace.NewAggregate(
			trace.ConvertSystemError(f.Close()),
			trace.ConvertSystemError(os.Remove(filePath)))
```

Reply:
Actually I'm not sure if the evaluation order is fully defined if you write it like that.
```suggestion
		// The file should be closed before removing it on Windows.
		closeErr := trace.ConvertSystemError(f.Close())
		removeErr := trace.ConvertSystemError(os.Remove(filePath))
		return trace.NewAggregate(closeErr, removeErr)
```

Reply:
Thanks, done.